config/config.go

```diff
@@ -40,8 +40,10 @@ type jsonFileQuery struct {
 }
 
 type jsonImplementation struct {
-	Code  jsonFileQuery `json:"code"`
-	Tests jsonFileQuery `json:"tests"`
+	Code                jsonFileQuery `json:"code"`
+	Tests               jsonFileQuery `json:"tests"`
+	CompilationDatabase string        `json:"compilationDatabase"`
+	ClangArguments      string        `json:"clangArguments"`
```

Comment:
Why this is not array?

Reply:
Just to make it easier for the user to write it as a regular string really. I can change it if you would prefer it as an array.